src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/ImageControls/When_Image.xaml

```diff
@@ -0,0 +1,56 @@
+﻿<Grid x:Class="Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls.ImageControls.When_Image"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:local="using:Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls.GridPages"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<StackPanel x:Name="test"
+				Spacing="10">
+		
+		<Border BorderBrush="Red"
+				BorderThickness="1"
+				Padding="2">
+			<Image x:Name="explicitRelativeNonRooted" x:FieldModifier="public">
+				<Image.Source>
+					<BitmapImage UriSource="Assets/Transitive-ingredient01.png" />
```

Comment:
in FrameworkElement_Opacity.xaml, you changed it to have it prefixed it with `/`, but not here
is that necessary or optional?

Reply:
The original code was assuming that any relative path with or without the "/" would search in the main assembly, which was incorrect. The lookup is now performed properly, which means that local (or explicit) lookup is required when done in runtime tests.